sensors/consts.py

```diff
@@ -0,0 +1,11 @@
+MEASUREMENT_TYPE_TEMPERATURE = 'temperature'
+MEASUREMENT_TYPE_HUMIDITY = 'humidity'
+MEASUREMENT_TYPE_SOUND = 'sound'
+MEASUREMENT_TYPE_PEOPLE_COUNT = 'people'
+
+MEASUREMENT_TYPE_CHOICES = (
+    (MEASUREMENT_TYPE_TEMPERATURE, 'Temperature'),
+    (MEASUREMENT_TYPE_HUMIDITY, 'Humidity'),
+    (MEASUREMENT_TYPE_SOUND, 'Sound'),
+    (MEASUREMENT_TYPE_PEOPLE_COUNT, 'People count'),
+)
```

Comment:
Please have a look at #80 for a neat(er) implementation of choice enums like this

Reply:
I'm not sure I see the point with that. It's semantically confusing having to rename types into "status", just to invoke a shorthand for CharField(max_length=100, choices=<something>). For me, this is less explicit and harder to read (as you have another level of indirection).